### ISSUE TITLE
T2I model uses absolute position embeddings and T2V model uses 1D+2D RoPE?

### ISSUE THREAD

**Issue by WarmCongee** [baaivision/NOVA#29]
Based on this [issue](https://github.com/baaivision/NOVA/issues/11#issuecomment-2676262551) and my understanding, does the t2i model in the model zoo use absolute position embeddings, while the t2v model uses 1D+2D RoPE?

**Comment by PhyscalX** [comment]
1. Fine-tuning absolute position embedding usually requires 1000 steps.

2. We will soon release our next-generation T2V models with 3D rotary embedding.

**Comment by WarmCongee** [comment]
@PhyscalX 
Hello, we first trained the absolute position embedding version based on this framework, and the results were very good. 

However, when we **switched to rope training** based on the **weights of the trained ape model,** we often encountered problems with excessively **high gradients or gradient explosions**, resulting in unstable training. 

We also discussed this with other authors, who pointed out that it may be necessary to freeze the layernorm of the input condition to avoid excessive scaling. We are very grateful for their guidance. However, this problem seems to still occur sometimes, so I would like to discuss it with you further. Have you encountered this situation before? Do you have any suggestions or experience for solving it?

**Comment by PhyscalX** [comment]
@WarmCongee 

We had adopted RoPE with an additional T2I training stage.
It should be more stable with fewer tokens, i.e., in a 256x256 T2I stage. 
Since we only released 512x512 T2I stage, you may initialize RoPE adoption from this weights.
